addons/hr_contract/base_action_rule.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.addons.base_action_rule.base_action_rule import get_datetime
+from openerp.fields import Datetime
```

Comment:
Use ```Odoo``` insted of ```openerp``` like : 
```
from odoo.fields import Datetime

```
If you fix issue because of your migration then changes will be possible into new api  :last_quarter_moon: